src/main/resources/log4j.properties

```diff
@@ -0,0 +1,16 @@
+# Root logger option
+log4j.rootLogger=INFO, file
+
+# Redirect log messages to console
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
+
+# Redirect log messages to a log file, support file rolling.
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.File=/Users/artem2/Mate_Academy/hibernate-cinema/logs/application.log
```

Comment:
/Users/artem2/...
is it the absolute path from your PC?

Reply:
nope, really bad ctrl+c strategy